src/oge/validation.py

```diff
@@ -471,16 +471,18 @@ def check_removed_data_is_empty(cems):
     """Checks that the rows removed by
     `data_cleaning.remove_cems_with_zero_monthly_data()` don't actually contain
     non-zero data"""
```

Comment:
Can you add typehint and docstring to this function. I think it would be good to update these every time we touch a function where it is missing